mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Socket.Mixfile do
   def project do
     [ app: :socket,
       version: "0.3.0",
-      elixir: "~> 1.0.0-rc1",
+      #elixir: "~> 1.1.1",
```

Comment:
A commit with mix stuff snuck in, please remove it so I can do the version upgrade myself (including publishing).